core/app/views/refinery/admin/_search.html.erb

```diff
@@ -1 +1,17 @@
-<%= render 'refinery/acts_as_indexed/admin/search', local_assigns if defined?(Refinery::ActsAsIndexed::Engine) -%>
```

Comment:
I think we should keep the partial as it and create a PR in https://github.com/refinery/refinerycms-acts-as-indexed/blob/master/app/views/refinery/acts_as_indexed/admin/_search.html.erb